textbook/12/2/normal.ipynb

```diff
@@ -0,0 +1,296 @@
+{
```

Comment:
<p>Just a question for myself, if <code>nsim</code> is 27, wouldn't the <code>for i in np.arange(nsim):</code> iterate over 1 through 26 (inclusive)? Is the intention of this loop to do 1 through 27? Just wanted to make sure I understand and that the function is doing what you intended.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>It actually starts at 0 to 26 so we would get the 27 elements.</p>